### DISCUSSION TITLE
Guition panel support

### DISCUSSION THREAD

**Discussion by kroon040** [Blackymas/NSPanel_HA_Blueprint#2349] [Ideas]
Hi
I bought an Guition panel, witch contains an esp32-s3. It is already supported with esphome and with lvgl support.

I already made a desing that looks like the ha_blueprint.

Maybe in future also support with this project?

![panel](https://github.com/user-attachments/assets/2fa44d69-ed14-47b9-aeb6-7405e00ddba2)

github page of my config

https://github.com/kroon040/HA-Panel-GUITION

**Comment by edwardtfn** [comment]
I have a similar panel and I'm thinking on start a project with it since LVGL support was released... It's really inspiring what you have done.
I wanna know how you compare the experience with a NSPanel. Not being limited by Nextion sounds very attractive, although I think I would miss it a bit when migrating to an entirely new platform.

**Reply by kroon040** [reply]
I also have 2 nspanels with the ha_blueprint. The guition panels are faster with the esp32s3 chip, and also more memory I think.

Switching between pages are more smooth than the nspanels. lvgl has more future and will be used more. This panel is also cheap, around 30 euros.

2 disadvantages , no physical buttons and I have to remove the back, and modded to get it build in the wall with boxes that are use here.

**Reply by fordcrews** [reply]
I have a couple of 5 and 7 inch waveshare panels with exp32's https://www.amazon.com/dp/B0DD7N19FT that I have been playing with.   Something cheaper, bigger, faster than the nspanels would really be nice.

**Reply by edwardtfn** [reply]
It's nice all those displays coming to the market and I feel the NSPanel is getting obsolete, however, I still to find something with the same professional finishing and as convenient as the NSPanel.
Lots of the solutions available are either developer boards or cheap plastic thing. 😩

**Reply by fordcrews** [reply]
My problem with NSPanel is it's price isn't dropping at all really.   With all the boards with faster processors, better displays, bigger displays, they should be dropping the price.    The cool thing with boards like the waveshare is you can easily print a 3d case for them or get someone on etsy to.

**Reply by edwardtfn** [reply]
Indeed
